src/app/merge-diff/merge-diff-container/merge-diff-container.component.scss

```diff
@@ -15,4 +15,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
-*/
\ No newline at end of file
+*/
+
+.container {
```

Comment:
I thought `container` just just a standard Bootstrap layout class, why is this created again? Or does it override some CSS properties?

Reply:
override for the resize to work